modules/apps/object/object-test/src/testIntegration/resources/com/liferay/object/web/internal/portlet/action/test/dependencies/test-bound-object-definitions.json

```diff
@@ -0,0 +1,661 @@
+				"indexedLanguageId": "",
+				"label": {
+					"en_US": "Create Date"
+				},
+				"localized": false,
+				"name": "createDate",
+				"objectFieldSettings": [
+				],
+				"readOnly": "true",
+				"readOnlyConditionExpression": "",
+				"required": false,
+				"state": false,
+				"system": true,
+				"type": "Date",
+				"unique": false
+			},
+			{
+				"DBType": "String",
+				"businessType": "Text",
+				"indexed": false,
+				"indexedAsKeyword": false,
+				"indexedLanguageId": "",
+				"label": {
+					"en_US": "External Reference Code"
+				},
+				"localized": false,
+				"name": "externalReferenceCode",
+				"objectFieldSettings": [
+				],
+				"readOnly": "false",
+				"readOnlyConditionExpression": "",
+				"required": false,
+				"state": false,
+				"system": true,
+				"type": "String",
+				"unique": false
+			},
+			{
+				"DBType": "Long",
+				"businessType": "LongInteger",
+				"indexed": true,
+				"indexedAsKeyword": true,
+				"indexedLanguageId": "",
+				"label": {
+					"en_US": "ID"
+				},
+				"localized": false,
+				"name": "id",
+				"objectFieldSettings": [
+				],
+				"readOnly": "true",
+				"readOnlyConditionExpression": "",
+				"required": false,
+				"state": false,
+				"system": true,
+				"type": "Long",
+				"unique": false
+			},
+			{
+				"DBType": "Date",
+				"businessType": "Date",
+				"indexed": false,
+				"indexedAsKeyword": false,
+				"indexedLanguageId": "",
+				"label": {
+					"en_US": "Modified Date"
+				},
+				"localized": false,
+				"name": "modifiedDate",
+				"objectFieldSettings": [
+				],
+				"readOnly": "true",
+				"readOnlyConditionExpression": "",
+				"required": false,
+				"state": false,
+				"system": true,
+				"type": "Date",
+				"unique": false
+			},
+			{
+				"DBType": "String",
+				"businessType": "Text",
+				"indexed": false,
+				"indexedAsKeyword": false,
+				"indexedLanguageId": "",
+				"label": {
+					"en_US": "Status"
+				},
+				"localized": false,
+				"name": "status",
+				"objectFieldSettings": [
+				],
+				"readOnly": "true",
+				"readOnlyConditionExpression": "",
+				"required": false,
+				"state": false,
+				"system": true,
+				"type": "String",
+				"unique": false
+			}
+		],
+		"objectFolderExternalReferenceCode": "default",
+		"objectLayouts": [
+		],
+		"objectRelationships": [
+			{
+				"deletionType": "cascade",
+				"edge": true,
+				"externalReferenceCode": "TESTOBJECTRELATIONSHIP1",
+				"label": {
+					"en_US": "Test Object Relationship 1"
+				},
+				"name": "testObjectRelationship1",
+				"objectDefinitionExternalReferenceCode1": "TESTOBJECTDEFINITION1",
+				"objectDefinitionExternalReferenceCode2": "TESTOBJECTDEFINITION2",
+				"objectDefinitionModifiable2": true,
+				"objectDefinitionName2": "TestObjectDefinition2",
+				"objectDefinitionSystem2": false,
+				"objectField": {
+					"DBType": "Long",
+					"businessType": "Relationship",
+					"externalReferenceCode": "TESTOBJECTRELATIONSHIP1OBJECTFIELD1",
```

Comment:
It is a detail, but I don't think we need "TESTOBJECTRELATIONSHIP1OBJECTFIELD1" since there can be only 1 object field per object relationship.